frustratometer/frustration/frustration.py

```diff
@@ -985,6 +989,11 @@ def write_tcl_script(pdb_file, chain, mask, distance_matrix, distance_cutoff, si
             }
             exit
         ''')
+    elif still_image_name:
```

Comment:
Here what is the expected behavior if both still_image_name and movie are defined?

Reply:
Expected behavior was what the code currently says, where it'll only try to do the still image if movie isn't defined. I was thinking I didn't want to have the vmd function doing too many things at once, but if we want to go in the direction of really developing the visualization capabilities in this function, I can think about how to organize that